File: createJson/src/main/java/com/ruzhan/jsonfile/model/Introduce.java
package com.ruzhan.jsonfile.model;

/**
 * Created by ruzhan123 on 2018/7/3.
 */
public class Introduce {

    public static final int TEXT = 1000;
    public static final int IMAGE = 1001;

    public int type;
    public String text;
    public String image;

    public Introduce(int type, String text, String image) {
        this.type = type;
        this.text = text;
        this.image = image;
    }
}

File: common/src/main/java/com/ruzhan/lion/model/Introduce.kt
package com.ruzhan.lion.model

import com.google.gson.annotations.SerializedName

/**
 * Created by ruzhan123 on 2018/7/5.
 */
data class Introduce (

    @SerializedName("type")
    var type: Int? = null,
    @SerializedName("text")
    var text: String? = null,
    @SerializedName("image")
    var image: String? = null
) {
    companion object {

        val TEXT = 1000
        val IMAGE = 1001
    }
}